crates/duty-tracker/src/contract_state_machine.rs

```diff
@@ -452,7 +452,8 @@ pub enum ContractState {
         l1_start_height: BitcoinBlockHeight,
     },
 
-    /// This state describes everything till the challenge transaction is confirmed.
+    /// This state describes everything from the momemnt the challenge transaction is confirmed to
+    /// the pre-assert transaction is confirmed.
```

Comment:
"This state describes everything from the moment the challenge transaction is confirmed to the moment the pre-assert transaction is confirmed."